### PR TITLE
batterpack implemented

### PR DESCRIPTION
**Pull request by AlexMdle**

### CODE REVIEW

File: src/main/java/thePackmaster/cards/batterpack/OpenBracket.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+                AbstractCard caard;
+
+                // Put attacks from group on top and draw.
+                for(int i = 0; i < OpenBracket.this.magicNumber; ++i) {
+                    if (!tmp.isEmpty()) {
+                        tmp.shuffle();
+                        caard = tmp.getBottomCard();
+                        tmp.removeCard(caard);
+                        if (p.hand.size() == 10) {
```
Comment (erasels) [inline src/main/java/thePackmaster/cards/batterpack/OpenBracket.java:57]: Don't check against 10, check against BaseMod.MAX_HAND_SIZE

**Review by erasels — CHANGES_REQUESTED** [review]

**Comment by erasels** [comment]
It's pretty cool to see how far you've come from the first pack to this. Barely anything I'd change, good job! (unless I missed a bunch of bad bugs)

File: src/main/java/thePackmaster/cards/batterpack/OpenBracket.java (hunk 1 above)
Comment (AlexMdle) [reply src/main/java/thePackmaster/cards/batterpack/OpenBracket.java:57]: Would preferd a high score of fixes needed but oh well.

Comment (AlexMdle) [reply src/main/java/thePackmaster/cards/batterpack/OpenBracket.java:57]: *of 0 fixes, goddammit

**Pull request merged**